kernel/src/device/pty/driver.rs

```diff
@@ -216,3 +224,10 @@ impl TtyDriver for PtyDriver {
         }
     }
 }
+
+impl Pollable for PtyDriver {
+    fn poll(&self, mask: IoEvents, poller: Option<&mut PollHandle>) -> IoEvents {
+        self.pollee
+            .poll_with(mask, poller, || self.check_io_events())
```

Comment:
The I/O events cannot be defined for the same pollee (with different meanings!) twice. The I/O events are already specified here:
https://github.com/asterinas/asterinas/blob/c292ccdccaf0c558ecad4f7b34b5dd15aeab6d15/kernel/src/device/pty/master.rs#L87-L94
https://github.com/asterinas/asterinas/blob/c292ccdccaf0c558ecad4f7b34b5dd15aeab6d15/kernel/src/device/pty/master.rs#L62-L84

```rust
     if self.slave().can_push() { 
         events |= IoEvents::OUT; 
     } 
```
```rust
        if self.output.lock().is_empty() {
            events |= IoEvents::OUT;
        }
```

The two code snippets will conflict with either other and cannot make sense.